### ISSUE TITLE
Strange Warning [Expected 'title' to be indentation at an of 2 but was found at 2]

### ISSUE THREAD

**Issue by FalkF** [rwjblue/ember-template-lint#332]
For this HBS:
```hbs
{{print-debug
  (hash
    client=environment.APP.version
    server=environment.apiVersion
    commitHash=commitHash
  )
  title="Version"
}}
```
I get following output:
```bash
  9:2  error  Incorrect indentation of attribute 'title' beginning at L9:C2. Expected 'title' to be indentation at an of 2 but was found at 2  attribute-indentation
```

Edit:
ember-template-lint does also seem to have a hard time detecting t.body or t.head from ember-light-table:
```hbs
...

  {{#t.body
    canExpand=true
    multiRowExpansion=false
  as |body|}}

...
```
```bash
  4:5  error  Incorrect indentation of positional param 't.body' beginning at L4:C5. Expected 't.body' to be indentation at an of 4 but was found at 5  attribute-indentation

```

**Comment by rwjblue** [comment]
Very strange indeed!  Would you mind submitting a failing test case (around [here](https://github.com/rwjblue/ember-template-lint/blob/master/test/unit/rules/lint-attribute-indentation-test.js#L51-L53)) to make it easier to track down?

**Comment by rwjblue** [comment]
Closed by #334
